src/registry.c

```diff
@@ -570,12 +570,10 @@ static inline PERSON *registry_person_get(const char *person_guid, time_t when)
         else {
             person_guid = buf;
             p = registry_person_find(person_guid);
-            if(!p) person_guid = NULL;
+            if(!p) p = registry_person_allocate(NULL, when);
```

Comment:
Oops, mistake here.
for line 573, cppcheck says: `Summary: Assignment of function parameter has no effect outside the function. Did you forget dereferencing it?`

So maybe remove `if(!p) person_guid = NULL;`?